src/handleLocalStorage.ts

```diff
@@ -1,14 +1,10 @@
-import { EventData } from "./types/main.js";
-
 const currentEventDataKey: string = "currentEventData";
 const savedEventsKey: string = "allSavedEvents";
 
-const storeDataInLocalStorage = (key: string, data: EventData | EventData[]) => {
+const storeDataInLocalStorage = (key: string, data: any) => {
```

Comment:
It is not `any`. Ideally we should see generic here. Same for `getDataFromLocalStorage` function. Do you recall the challenge that data type should depend on the value of the key?